Flaky test  - mainline multi-host

The mainline test is failing intermittently (maybe one in ten runs). This is annoying and should be fixed
```
======================================================================
FAIL: Run a mainline multi-host test.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tests/st/libnetwork/test_mainline_multi_host.py", line 96, in test_multi_host
    workload_h2n1_1])
  File "/tests/st/utils/utils.py", line 122, in wrapped
    raise e
AssertionError: workload_h1n1_1 cannot ping 10.0.0.2
```

--- a/tests/st/libnetwork/test_mainline_multi_host.py
+++ b/tests/st/libnetwork/test_mainline_multi_host.py
@@ -91,7 +91,8 @@ class MultiHostMainline(TestBase):
             # 1, and not those on network 2.  Ping using IP for all workloads,
             # and by hostname for workloads on the same network (note that
             # a workloads own hostname does not work).
-            self.assert_connectivity(pass_list=[workload_h1n1_1,
+            self.assert_connectivity(retries=2,
+                                     pass_list=[workload_h1n1_1,
                                                 workload_h1n1_2,
                                                 workload_h2n1_1])
             # TODO: docker_gwbridge iptable FORWARD rule takes precedence over

